pkg/gatherers/clusterconfig/container_images.go

```diff
@@ -202,3 +205,18 @@ func gatherImages(startMonth string, img2month2count img2Month2CountMap, contain
 		}
 	}
 }
+
+// obfuscateContainerEnvVars finds env variables within the given container list
+// and, if they are a target, it will obfuscate their value
+func obfuscateContainerEnvVars(containers []corev1.Container) {
+	targets := []string{"HTTP_PROXY", "HTTPS_PROXY"}
+	search := regexp.MustCompile(strings.Join(targets, "|"))
```

Comment:
Nit: I think regex pattern is not required or is it? On the other hand it's probably not a big deal....

Reply:
I accept the nit.

Actually, this way is not very costly. Regex is not using wildcards. Only check for exact notation is usually very fast. I did the code this way just in case more ENV VARS are suitable to be obfuscated in the future: you only need to add a new item to the slice.

If you prefer not involve any regexp, what is fair, I can reuse the slice approach and make a loop comparing strings with Env var names. Or, if you know any other magic or vendor package already on the project that could do this easier, please recommend.

Reply:
OK fair point. Let's leave it as is